cypress/e2e/GUI/testGui.cy.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types="Cypress" />
+
+describe('Testando a aplicação EngageSphere', () => {
+
+    // Executa antes de cada teste
+    beforeEach(() => {
+      // Define uma data e hora fixa
+      const fixedDate = new Date(2024, 6, 10); // 10 de julho de 2024
+      cy.clock(fixedDate.getTime());
+      cy.visit('http://localhost:3000');
```

Comment:
Visto que você definiu a `baseUrl` no arquivo `cypress.config.js`, aqui você não precisa repetí-la.

```suggestion
      cy.visit('');
```